config/dynamic-scaling-configuration.yaml

```diff
@@ -3,10 +3,31 @@
 #The following properties must be defined for each instance types compute node configuration:
 #   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
 
+# For each supported cloud provider, the following properties must be provided for the machine type
```

Comment:
Some thought about this config file. What happens if we start having some configuration specific per region?

Reply:
The config will need to be enhanced or think about combing this part with the providers configuration.

Reply:
:+1: